lib/lens.ex

```diff
@@ -17,12 +17,14 @@ defmodule Lens do
   end
 
   @doc ~S"""
-  Returns a lens that focuses on the whole data.
+  Returns a lens that focuses on all of the data in the given scope.
```

Comment:
I don't really like this "scope" - I don't think there is a mention of "scope" anywhere else in the doc. "all of the" vs. "the whole" is a good change, I think.